src/Illuminate/Queue/Console/RetryCommand.php

```diff
@@ -53,7 +52,7 @@ protected function getJobIds()
         $ids = (array) $this->argument('id');
 
         if (count($ids) === 1 && $ids[0] === 'all') {
-            $ids = Arr::pluck($this->laravel['queue.failer']->all(), 'id');
+            $ids = $this->laravel['queue.failer']->all();
```

Comment:
The method is named "getJobIds". The variable is named "ids". Isn't it weird that it now contains full database records?

Reply:
no. the result of `$this->laravel['queue.failer']->all()` were all fields from the failed_tables table converted to an array. Now it's only the 'id' fields from the failed_tables table. So no need the pluck the ids.